src/main/resources/db/migration/v0.18.0.5__hpterm.sql

```diff
@@ -13,9 +13,9 @@ CREATE TABLE public.hpterm_aud (
 ALTER TABLE ONLY public.hpterm
     ADD CONSTRAINT hpterm_pkey PRIMARY KEY (curie);
 ALTER TABLE ONLY public.hpterm
-    ADD CONSTRAINT hpterm_curie_fk FOREIGN KEY (curie) REFERENCES public.phenotypeterm(curie);
```

Comment:
Ah, I see. Will update. @cmpich suggested `setLoadOnlyIRIPrefix` and it seemed to work.

Reply:
It might... but its better to use the default namespace... if there is one.

Reply:
On this migration it will fail because hpterm_curie_fk already exists in the database. So you have to remove it via a migration and then add it back. In other words:
`DROP CONSTRAINT hpterm_curie_fk`
`ADD CONSTRAINT hpterm_curie_fk FOREIGN KEY (curie) REFERENCES public.ontologyterm(curie);`

Reply:
Actually, maybe the sql file on alpha is fine and the Java class needs to change. Maybe HPTerm needs to extend PhenotypeTerm instead of OntologyTerm?

Reply:
Sure thats fine... but once a migration has been applied. We can't edit it, because it will not be rerun. So a new migration has to be created to do this change.

Reply:
Does HPTerm extend Phenotype? If so then this migration doesn't need to happen. But if not then the HP terms need to get deleted from the pheno ontology table and the this migration needs to happen.

Reply:
I think this was resolved in Daily Scrum, but just to make sure:

In the LinkML model:
```
  HPTerm:
    is_a: PhenotypeTerm
```
So, it's the HPTerm entity class that needs to change to inherit from PhenotypeTerm, rather than the migration changing.